build-gluster-org/jobs/centos7-regression.yml

```diff
@@ -6,36 +6,63 @@
     concurrent: true
 
     scm:
-    - git:
-        branches:
-        - $GERRIT_BRANCH
-        refspec: $GERRIT_REFSPEC
-        choosing-strategy: gerrit
-        url: git://review.gluster.org/glusterfs.git
-        wipe-workspace: false
+    - glusterfs
 
     properties:
+    - github:
+        url: https://github.com/gluster/glusterfs
     - discard-after-x:
         x: 90
     - one-build-per-node
 
     triggers:
-    - gerrit:
-        server-name: review.gluster.org
-        trigger-on:
-          - comment-added-contains-event:
-              comment-contains-value: "recheck centos"
-          - comment-added-event:
-              approval-category: Verified
-              approval-value: 1
-        failure-message-file: gluster_regression.txt
-
-        projects:
-          - project-compare-type: 'PLAIN'
-            project-pattern: 'glusterfs'
-            branches:
-              - branch-compare-type: 'ANT'
-                branch-pattern: '**'
+    - github-pull-request:
+        cancel-builds-on-update: true
+        allow-whitelist-orgs-as-admins: true
+        admin-list:
```

Comment:
Could we add a comment here on how we get the list, or how do people get added ? 

I would prefer to have that tied to github team, but I guess that's not doable for now.